frontend/src/common/enums/ui/button-style.enum.ts

```diff
@@ -1,6 +1,7 @@
 enum ButtonStyle {
   FILLED = 'Filled',
   OUTLINED = 'Outlined',
+  GRAY = 'Gray',
```

Comment:
this is a color, not style. Create a separate enum `ButtonColor` for it. This enum should have two colors - orange and gray